### ISSUE TITLE
[BUG] --mix-str will cause "<lambda>() takes 0 positional arguments but 1 was given" if the code has an async decorator

### ISSUE THREAD

**Issue by null-routed** [dashingsoft/pyarmor#1101]
Obfuscating python code containing an async decorator using "--mix-str" will cause the following error: 
```
File "<frozen __main__>", line 3, in <module>
File "<frozen test>", line 13, in <module>
File "<frozen test>", line 4, in my_decorator
TypeError: <lambda>() takes 0 positional arguments but 1 was given
```
**How to reproduce**

```python
import asyncio

def my_decorator(func):
    async def wrapper(*args, **kwargs):
        try:
            result = await func(*args, **kwargs)
            return "success"
        except Exception as e:
            return "failure"
    return wrapper

@my_decorator
async def my_func():
    # do some actually async stuff
    return 1

if __name__ == "__main__":
    loop = asyncio.get_event_loop()
    loop.run_until_complete(my_func())
```

`pyarmor gen --mix-str foo.py`.
You can tell the error refers to line 4, which is where the wrapper is defined. However the code runs fine if I make the wrapper not async.

**Comment by jondy** [comment]
Got it, thanks, I'll check it.

**Comment by jondy** [comment]
Fixed in `8.1.dev7`
